### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by murissaab1976**
Updated README.md version 1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,56 @@
...
+
+The project encompasses data preparation, exploratory data analysis, model training, and implementation for real-world application. We utilized AWS SageMaker for model development and training, focusing on creating a regression model optimized for minimizing RMSE.
+
+## Recommendations
+
+Based on our findings, we recommend prioritizing resource allocation to zip codes identified as high-risk due to socioeconomic disparities. Implementing our model's predictions will enable a more responsive and equitable healthcare crisis management approach.
+
+## Repository Structure
+
+- `data/`: Contains datasets used for analysis.
```
Comment (shailja-somani-0) [inline README.md]: I don't believe this repository structure matches ours - could you update it to do so?

File: README.md [hunk 2]
```diff
@@ -1,4 +1,56 @@
...
+- `notebooks/`: Jupyter notebooks detailing our data analysis and modeling process.
+- `src/`: Source code for data preprocessing, modeling, and analysis.
+- `design`: Design Documentation **(assuming the final design doc goes in this folder)**
+
+## Getting Started
+
+To replicate our analysis or explore the datasets:
+
+1. Clone the repository.
+2. Navigate to the `notebooks/` directory to find Jupyter notebooks with detailed analysis steps.
```
Comment (shailja-somani-0) [inline README.md]: Again, I don't believe this is our repo structure! Our datasets are in the folder `dataset_csvs` & the notebooks must be run from within SageMaker as follows: (1) [team2-ads508.ipynb](https://github.com/shailja-somani-0/ADS-508-Team-2/blob/main/team2-ads508.ipynb) (2) [SageMakerAutopilotCandidateDefinitionNotebook.ipynb](https://github.com/shailja-somani-0/ADS-508-Team-2/blob/main/SageMakerAutopilotCandidateDefinitionNotebook.ipynb)

File: README.md [hunk 3]
```diff
@@ -1,4 +1,56 @@
...
+1. [Maryland COVID-19 Cases by Zip Code](https://opendata.maryland.gov/Health-and-Human-Services/MD-COVID-19-Cases-by-ZIP-Code/ntd2-dqpx/about_data)
+2. [Social Determinants of Health Database](https://www.ahrq.gov/sdoh/data-analytics/sdoh-data.html#download)
+3. [PLACES: Local Data for Better Health](https://data.cdc.gov/500-Cities-Places/PLACES-Local-Data-for-Better-Health-ZCTA-Data-2023/qnzd-25i4/about_data)
+4. [Maryland Census Data](https://data.imap.maryland.gov/datasets/eb706b48117b43d482c63d02017fc3ff/explore?location=38.795704%2C-77.268400%2C7.90)
+5. [ZTCA to Zip Code Crosswalk](https://udsmapper.org/zip-code-to-zcta-crosswalk/)
+
+
+## Methodology
+
+The project encompasses data preparation, exploratory data analysis, model training, and implementation for real-world application. We utilized AWS SageMaker for model development and training, focusing on creating a regression model optimized for minimizing RMSE.
```
Comment (shailja-somani-0) [inline README.md:39]: Can we expand upon the use of DataWrangler, resulting ensemble model, & its RMSE here? From the Model Training section of the data sci design document. Just  approx 2 sentences should be fine!

File: README.md [hunk 4]
```diff
@@ -1,4 +1,56 @@
-# ADS 508 Final Project
-Team Members: Shailja Somani, John Vincent Deniega, & Muris Saab\
-Spring 2024\
-University of San Diego
+# ADS-508 Team Project
```
Comment (shailja-somani-0) [inline README.md]: I think we should still keep team member names up here! (I saw you deleted the previous header, but at least names are still useful I think)

**Review by shailja-somani-0 — CHANGES_REQUESTED** [review]
Looks good overall, but a few changes requested, esp bc I don't think the repo structure you wrote out matches our repo. Thanks!

File: README.md (hunk 4 above)
Comment (murissaab1976) [reply README.md]: done

File: README.md (hunk 3 above)
Comment (murissaab1976) [reply README.md:39]: done

File: README.md (hunk 1 above)
Comment (murissaab1976) [reply README.md]: Good catch, I think given the simple and small structure, I will remove this section if that's ok. It is pretty self explanatory. Thoughts?

File: README.md (hunk 2 above)
Comment (murissaab1976) [reply README.md]: fixed

**Review by murissaab1976 — COMMENTED** [review]
updated

**Pull request merged**